x-pack/plugins/timelines/public/components/t_grid/helpers.tsx

```diff
@@ -388,3 +406,60 @@ export const getDefaultViewSelection = ({
 
 /** This local storage key stores the `Grid / Event rendered view` selection */
 export const ALERTS_TABLE_VIEW_SELECTION_KEY = 'securitySolution.alerts.table.view-selection';
+
+const buildISQueryMatch = ({
```

Comment:
consider adding unit tests for this function, and the `buildEXISTSQueryMatch` and `buildEXISTSQueryMatch` functions below

Reply:
Done!